llvm/utils/mlgo-utils/mlgo/corpus/flags.py

```diff
@@ -0,0 +1,22 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Library functions for setting up common parser arguments"""
+
+from argparse import ArgumentParser
+
+def add_verbosity_arguments(parser: ArgumentParser) -> None:
+    """Adds the arguments for verbosity to the ArgumentParser
+
+    Arguments:
+        parser -- the argument parser being modified with verbosity arguments
```

Comment:
```suggestion
        parser: The argument parser being modified with verbosity arguments.
```
Nit.

`:` instead of `--`, and then a full sentence for the description.